nvflare/app_common/workflows/cyclic_ctl.py

```diff
@@ -150,19 +155,48 @@ def _process_result(self, client_task: ClientTask, fl_ctx: FLContext):
         # will get the updated shareable
         task = client_task.task
 
-        # update the global learnable with the received result (shareable)
-        # e.g. the received result could be weight_diffs, the learnable could be full weights.
-        self._last_learnable = self.shareable_generator.shareable_to_learnable(client_task.result, fl_ctx)
+        result = client_task.result
+        if isinstance(result, Shareable):
```

Comment:
if result is not a Shareable do we need to log any errors or raise Exceptions?

Reply:
We could, but this is only for the purpose of getting return code in case EARLY_TERMINATION is received. In theory (though not in reality) a non-Shareable type of object could be received and still be valid.